header.cmake

```
@@ -316,9 +316,10 @@ macro(HEADER_INSTALL)
     string(REGEX REPLACE "${CMAKE_BINARY_DIR}" "" DIR "${DIR}")
     string(REGEX REPLACE "${PROJECT_SOURCE_DIR}" "" DIR "${DIR}")
     string(REGEX REPLACE "include(/|$)" "" DIR "${DIR}")
+    cmake_path(SET INSTALL_PATH NORMALIZE "${CMAKE_INSTALL_INCLUDEDIR}/${DIR}")
```

```suggestion
    if(CMAKE_VERSION` VERSION_GREATER 3.20)
      # workaround CMP0177
      cmake_path(SET INSTALL_PATH NORMALIZE "${CMAKE_INSTALL_INCLUDEDIR}/${DIR}")
    else()
      set(INSTALL_PATH "${CMAKE_INSTALL_INCLUDEDIR}/${DIR}")
    endif()
```

As https://cmake.org/cmake/help/latest/command/cmake_path.html#command:cmake_path was added in 3.20, and we don't really fear breaking things for versions < 3.20